ui/eslint.config.js

```diff
@@ -10,6 +10,16 @@ export default [
   {
     plugins: {
       pluginReactConfig: react,
-    }
+    },
+  },
+  {
+    rules: {
+      'no-empty': 'off',
+      'no-useless-escape': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+    },
+  },
+  {
+    ignores: ['**/i18n/strings/*','**/G6register.tsx','**/.umi/*'],
```

Comment:
Add a `'**/.umi-production/*'` here to ignore manifests generated when building the site.

Reply:
Are they also checked when building?

Reply:
Yes, folder `.umi-production` will be created if you run once `yarn build`

Reply:
get